app/controllers/content/index.js

```diff
@@ -78,6 +80,15 @@ export default Ember.Controller.extend(Analytics, {
             this.get('model.dateCreated');
     }),
 
+    editButtonLabel: Ember.computed('model.provider.reviewsWorkflow', 'model.reviewsState', function () {
+        const edit_preprint = 'content.project_button.edit_preprint';
+        const edit_resubmit_preprint = 'content.project_button.edit_resubmit_preprint';
+        return (
+            this.get('model.provider.reviewsWorkflow') === PRE_MODERATION
+            && this.get('model.reviewsState') == REJECTED
+        ) ? edit_resubmit_preprint : edit_preprint
```

Comment:
`;`

(or maybe replace this ternary with an `if`, since it's long enough to wrap, but that's just a matter of taste)